.gitignore

```diff
@@ -0,0 +1,43 @@
+# Whisk
```

Comment:
I suggest one git ignore for the repo. There are elements in here that are not necessary.